docs/development.adoc

```diff
@@ -158,34 +158,25 @@ $ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--ena
 
 * `odo` and `oc` binaries in `$PATH`.
 
-.Procedure:
-
-To deploy an integration test:
+.How to write:
 
+Refer to the odo clean test link:https://github.com/openshift/odo/blob/master/tests/template/template_cleantest_test.go[`template`].
 
-* For the entire integration suite use:
-----
-$ make test-e2e
```

Comment:
@amitkrout again, you're removing documentation and not adding it again... no documentation on `make test-integration`

Reply:
Ahh...my bad. Will update it

Reply:
https://github.com/openshift/odo/pull/1797/files#diff-c26dca7a0015d5429db3d986bc160252R183